components/uaa-activator/Gopkg.toml

```diff
@@ -0,0 +1,36 @@
+[[constraint]]
```

Comment:
the time will come that we will have to start using modules :)

Reply:
not yet :(